### PR TITLE
docs: Vertical Guidance Advanced Guide

### LINKED ISSUE #500: Advanced Guide VNAV

### PR DESCRIPTION
**Pull request by frankkopp**
Closes #500 

## Summary
Advanced Pilot's Guide for Vertical Guidance

- [x] Overview
- [x] Selected Vertical Modes
 		- [x] OP CLB
		- [x] OP DES
		- [x] V/S and FPA
		- [x] Expedite
- [x] Managed Vertical Modes
		- [x] CLB
		- [x] DES
- [x] Speed/Mach Control
- [x] Navigation Display Symbols
- [x] Primary Flight Display Indications
- [x] Example Managed Flight

### Location
https://docs-git-fork-frankkopp-adv-guide-vnav-flybywire.vercel.app/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/overview/

Discord username (if different from GitHub): Cdr_Maverick#6475

### CODE REVIEW

**Comment by Valastiri** [comment]
Added a few organizational things + new section **symbology**

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/selected-modes.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+Autothrust is active it sets the thrust to CLB THRUST automatically.
+
+When the aircraft is in EXP DES, the target speed is 340 kt or 0.8 Mach which is maintained with pitch control.
+If Autothrust is active it sets the thrust at IDLE automatically.
+
+When EXPEDITE is engaged, the system disregards speed constraints (SPD CSTR), altitude constraints (ALT CSTR), and
+speed limits (SPD LIM).
+
+EXPEDITE can be engaged when
+‐ the aircraft has been in flight for more than 5 s
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/selected-modes.md:121]:
```suggestion

‐ the aircraft has been in flight for more than 5 s
```
Newline needed for proper rendering

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/selected-modes.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+‐ managed speed is available.
+
+It is engaged manually only when
+‐ the FCU selected altitude is higher than present altitude, EXP CLB mode engages
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/selected-modes.md:126]:
```suggestion

‐ the FCU selected altitude is higher than present altitude, EXP CLB mode engages
```
Newline needed for proper rendering

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+The SRS mode engages automatically when the thrust levers are set to the TOGA or FLX/MCT detent (when a Flex
+takeoff temperature has been selected), if:
+
+- V2 has been inserted in the MCDU PERF TAKEOFF page
+- the slats are extended
+- the aircraft has been on ground for at least 30 seconds
+
+The SRS mode disengages:
+- automatically, at the acceleration altitude (ACC ALT), or if ALT* or ALT CST* mode engages (above 400 ft RA)
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md:24]:
```suggestion

- automatically, at the acceleration altitude (ACC ALT), or if ALT* or ALT CST* mode engages (above 400 ft RA)
```
Newline needed for proper rendering

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+    - the acceleration altitude defined in the MCDU PERF TO or GA page is below the lowest altitude constraint and
+      also below the altitude selected in the FCU
+- In flight in climb or go-around phase
+    - if lateral navigation (NAV) is engaged
+    - if the altitude selected in the FCU is above the current altitude and the aircraft captures or flies an
+      altitude constraint
+
+The CLB mode can be engaged when:
+
+‐ the aircraft has been in flight for more than 5 seconds
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md]:
```suggestion
- the aircraft has been in flight for more than 5 seconds
```
Wrong dash style used, messes up rendering

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+
+- Idle path segment:
+    - Autothrust uses idle thrust while the AF/FD controls speed by adjusting the vertical trajectory (pitch)
+- Geometric path segment:
+    - AP/FD controls the required vertical path while Autothrust controls speed.
+- Re-pressurization segment:
+    - If required, this ensures a specific rate of pressurization for the cabin during descent. It is calculated
+      from the destination airport altitude and the selected cabin rate (default -350ft/min which can be modified)
+
+  !!! warning "Re-pressurization segments are not yet implemented in the A32NX"
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md]:
```suggestion
!!! warning "Re-pressurization segments are not yet implemented in the A32NX"
```
Remove leading spaces or it isn't rendered as a warning it seems.

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+
+If the aircraft levels off at an ALT CSTR, the DES mode arms and remains armed until the aircraft passes the
+constraint, then reengages (if the FCU altitude is set below the altitude of the constraint).
+
+### Engaging DES Mode
+The DES mode can be engaged when:
+
+- the altitude selected in the FCU os lower than the present altitude
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md]:
```suggestion
- the altitude selected in the FCU is lower than the present altitude
```

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+- either NAV, LOC^*^, LOC are active
+- takeoff, climb or go-around phase are not active
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md:150]: Isn't this duplicating some information from 2 lines lower (`not in TO, G/S, LAND, FINAL or GA mode`), should it be combined?

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md [hunk 8]
```diff
@@ -0,0 +1,232 @@
...
+aircraft is at or beyond the top of descent point it descents at idle thrust.
+
+After initiating the descent the aircraft shows a vertical deviation symbol (green dot) next to the altitude scale on the
+PFD so that the flight crew can monitor the aircraft's position relative to the descent path.
+
+When the speed is managed, a target speed range is displayed in the speed band on the PFD (magenta bars) which defines
+acceptable speed variations around the nominal descent speed target.
+
+Associated with the VDEV displayed on PFD, the ND shows an intercept point (zigzag symbol) on the flight plan. It
+indicates the position where the system predicts that the aircraft will intercept the descent profile.
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md:178]: Might be good to also highlight VDEV is shown in the PROG page of the MCDU?

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md [hunk 9]
```diff
@@ -0,0 +1,232 @@
...
+If the aircraft is below the descent profile, its speed will be maintained at target speed until it reaches the
+descent profile. The lower margin becomes effective when the aircraft is on the descent profile but has to lose
+speed in order to stay on it.
+
+!!! warning "The MODE DRAG message is not yet implemented in the A32NX"
+
+## Approach modes
+
+### G/S, G/S*
+Glideslope capture (G/S^*^) and glideslope modes are used in precision approaches (ILS, MLS).
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md]:
```suggestion
Glideslope capture (G/S^*^) and glideslope (G/S) modes are used in precision approaches (ILS, MLS).
```

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/speed-control.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
+# Speed/Mach Control
+In flight, either the AP/FD pitch control, or Autothrust may acquire and hold a target speed or Mach
+number, depending on the engaged modes.
+
+Speed control is:
+‐ Managed when the target comes from the FMGS
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/speed-control.md:7]:
```suggestion

‐ Managed when the target comes from the FMGS
```
Newline needed for proper rendering

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/speed-control.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+
+The speed profile when NAV is not enabled is as follows: V2, SPD LIM, ECON CLB SPD/MACH, ECON CRZ MACH, ECON or
+preset DES MACH/SPD, SPD LIM - V~APP~.
+
+In managed mode the AP and FD pitch modes can control a target SPD/MACH or a vertical trajectory, and the A/THR
+mode can control a fixed thrust or a target SPD/MACH. However, the AP/FD and the A/THR cannot
+both control a target SPD/MACH simultaneously.
+
+Therefore, the AP/FD pitch modes and A/THR mode are coordinated as follows:
+‐ If an AP/FD pitch mode controls a vertical trajectory, the A/THR mode controls the target SPD/MACH.
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/speed-control.md:37]:
```suggestion

‐ If an AP/FD pitch mode controls a vertical trajectory, the A/THR mode controls the target SPD/MACH.
```
Newline needed for proper rendering

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md [hunk 12]
```diff
@@ -0,0 +1,232 @@
...
+After initiating the descent the aircraft shows a vertical deviation symbol (green dot) next to the altitude scale on the
+PFD so that the flight crew can monitor the aircraft's position relative to the descent path.
+
+When the speed is managed, a target speed range is displayed in the speed band on the PFD (magenta bars) which defines
+acceptable speed variations around the nominal descent speed target.
+
+Associated with the VDEV displayed on PFD, the ND shows an intercept point (zigzag symbol) on the flight plan. It
+indicates the position where the system predicts that the aircraft will intercept the descent profile.
+
+See all symbols below: [Vertical Guidance Symbology](#vertical-guidance-symbology).
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md]:
```suggestion
See all symbols below: [Vertical Guidance Symbology](nd-symbols.md).
```
Think this had the wrong link?

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+|   ![](../../../assets/advanced-guides/vnav/symbols/leveloff-climb-cyan.png){loading=lazy}    | Indicates *top of climb*                                   | The FCU selected altitude (cyan)                                                                                                                                                   |
+|  ![](../../../assets/advanced-guides/vnav/symbols/leveloff-climb-magenta.png){loading=lazy}  |                                                            | A constraint altitude (if all appropriate modes are engaged) (magenta)                                                                                                             |
+|                                                                                              |                                                            | Will not appear if the aircraft is 100 ft above or below selected altitude                                                                                                         |
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md:7]: Should this be in a different row? It looks strange as a row in a table with no icon, it doesn't really indicate it belongs to the previous icon to me.

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+|  ![](../../../assets/advanced-guides/vnav/symbols/leveloff-descent-cyan.png){loading=lazy}   | Indicates *level-off*                                      | The FCU selected altitude (cyan)                                                                                                                                                   |
+| ![](../../../assets/advanced-guides/vnav/symbols/leveloff-descent-magenta.png){loading=lazy} |                                                            | A constraint altitude (if all appropriate modes are engaged) (magenta)                                                                                                             |
+|                                                                                              |                                                            | Will not appear if the aircraft is 100 ft above or below selected altitude                                                                                                         |
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md:10]: Same as the other, looks a bit strange as part of a new row? (given there's row borders in the table)

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+|     ![](../../../assets/advanced-guides/vnav/symbols/intercept-white.png){loading=lazy}      | Indicates *intercept point*                                | Symbol is White if NAV mode is engaged                                                                                                                                             |
+|      ![](../../../assets/advanced-guides/vnav/symbols/intercept-cyan.png){loading=lazy}      |                                                            | Symbol is Blue, if DES mode is engaged                                                                                                                                             |
+|                                                                                              |                                                            | If vertical deviation is detected while in DES mode, continuously indicates predicted point where aircraft will intercept the descent path                                         |
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md:20]: Same remark as before: it is a little weird to see this as a separate row in a table.

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+|                                                                                              |                                                            |                                                                                                                                                                                    |
+|       ![](../../../assets/advanced-guides/vnav/symbols/speed-change.png){loading=lazy}       | Indicates a *speed change*                                 | Point on the flight plan where the aircraft with automatically accelerate or decelerate to a new computed speed                                                                    |
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md]:
```suggestion
|       ![](../../../assets/advanced-guides/vnav/symbols/speed-change.png){loading=lazy}       | Indicates a *speed change*                                 | Point on the flight plan where the aircraft will automatically accelerate or decelerate to a new computed speed                                                                    |
```

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+|        ![](../../../assets/advanced-guides/vnav/symbols/cstr-white.png){loading=lazy}        |                                                            | Symbol is White indicates `ALT CSTR` is not applicable and NAV mode is engaged                                                                                                     |
+|                                                                                              |                                                            |                                                                                                                                                                                    |
+|     ![](../../../assets/advanced-guides/vnav/symbols/time-marker.png){loading=lazy} hhmm     | Indicates TIME MARKER or EQUI-TIME POINT (ETP)             | Symbol is appears in Green indicating on the flight plan where the aircraft will reach the time marker or ETP                                                                      |
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md]:
```suggestion
|     ![](../../../assets/advanced-guides/vnav/symbols/time-marker.png){loading=lazy}          | Indicates TIME MARKER or EQUI-TIME POINT (ETP)             | Symbol appears in Green indicating on the flight plan where the aircraft will reach the time marker or ETP                                                                      |
```

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md [hunk 18]
```diff
@@ -0,0 +1,37 @@
...
+|                                                                                              |                                                            |                                                                                                                                                                                    |
+|      ![](../../../assets/advanced-guides/vnav/symbols/energy-circle.png){loading=lazy}       | Indicates the ENERGY CIRCLE symbol                         | Symbol is oriented to the current track line and centered on the aircraft position                                                                                                 |
+|                       (Energy Circle not yet implemented in the A32NX)                       |                                                            | Indicates Required Distance to Land                                                                                                                                                |
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md:34]: Should this be a note instead?

Also same remark, it feels a little strange of not having them in one row.

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/example.md [hunk 19]
```diff
@@ -0,0 +1,194 @@
...
+    ![Winds Example](../../../assets/advanced-guides/vnav/winds.png){loading=lazy width=60% align=left}
+
+    The winds page accept the following format: `HDG / SPEED / ALTITUDE`. 
+
+    When entering the winds for the alternate destination the flight crew can omit the `ALTITUDE` value as that has been populated on the right side of the screen. 
+
+To enter the examples highlighted in the image you can type:
+
+- `050/009/390` and press LSK 1L to input the FL390 winds.
+- `081/012` and press LSK 1R to input the alternate destination winds at FL100. **Note:** the flight level is omitted in the entry.
```
Comment (pdellaert) [inline docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/example.md:194]: Do we want to highlight Wind Request can also be used?

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md (hunk 7 above)
Comment (frankkopp) [reply docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/managed-modes.md:150]: No, this is the flight phase, the one below the guidance mode.

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md (hunk 13 above)
Comment (frankkopp) [reply docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md:7]: MD tables are rather limited - if you find a better way to layout this please go ahead.

Comment (pdellaert) [reply docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md:7]: Yeah, I guess there's some limitations... The only solution seems to be by using literal HTML tags (`<br />`), which isn't really a clean way either.

File: docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/example.md (hunk 19 above)
Comment (frankkopp) [reply docs/pilots-corner/advanced-guides/flight-guidance/vertical-guidance/example.md:194]: We can do this later - don't think we have a guide for that import yet.

**Pull request merged**